README.md

```diff
@@ -1,61 +1,436 @@
 # Argo Toolbox API
 
-## Docker compose
+A step‑by‑step guide to install, run, and explore the **Toolbox API** locally.  
+This documentation is tailored for **non‑technical users**, with simple instructions and no prior Docker experience required.
 
-This Docker Compose configuration allows you to run the Argo Toolbox API locally on your laptop, replacing the Kubernetes deployment for easier local development and testing.
+---
 
-### Prerequisites
+## 🚀 Quick start
 
-- Docker (Docker Desktop for Windows)
+If you already have Docker and Git configured on your local machine, you can just follow these steps:
```

Comment:
Rather than add extra instructions here, I'd suggest we give a link to skip down to the "Clone the Repository" section.